src/procedures/RevokeKyc.ts

```diff
@@ -5,9 +5,20 @@ import { ProcedureType, PolyTransactionTag, ErrorCode, RevokeKycProcedureArgs }
 import { PolymathError } from '../PolymathError';
 import { Shareholder, SecurityToken } from '../entities';
 
+/**
+ * Procedure that revoke the KYC validation of a list of investors
+ */
 export class RevokeKyc extends Procedure<RevokeKycProcedureArgs, Shareholder[]> {
   public type = ProcedureType.RevokeKyc;
 
+  /**
+   * - Disallows the ability of purchase and sale for each investor in the list. Also makes invalid the KYC
```

Comment:
No hyphen

Sets all KYC dates for a shareholder to epoch. This effectively makes them unable to send or receive Security Tokens